tests/nn_test.py

```diff
@@ -81,6 +101,10 @@ def testSoftplusValue(self):
     val = nn.softplus(89.)
     self.assertAllClose(val, 89., check_dtypes=False)
 
+  def testSquareplusValue(self):
+    val = nn.squareplus(1e3)
+    self.assertAllClose(val, 1e3, check_dtypes=False)
```

Comment:
I think you need to adjust tolerances here: for float64 the default is 1E-15, and the expected output differs by more than that.